src/components/new-safe/steps/Step0/index.tsx

```diff
@@ -56,8 +56,8 @@ export const ConnectWalletContent = ({ onSubmit }: { onSubmit: StepRenderProps<N
   )
 }
 
-const CreateSafeStep0 = ({ onSubmit, onBack }: StepRenderProps<NewSafeFormData>) => {
-  const { isConnected } = useCreateSafe()
+const CreateSafeStep0 = ({ onSubmit, onBack, setStep }: StepRenderProps<NewSafeFormData>) => {
+  const { isConnected } = useCreateSafe(setStep)
```

Comment:
I tried using a context but since it doesn't do anything except call `useIsConnected` and return its value it seems a bit overkill as opposed to calling `useIsConnected` in 4 components.

Reply:
Alright. Maybe we can reassess this when we review everything together. I know that there was discussion in the widget PR about using context.